### ISSUE TITLE
The final argument to magicString.overwrite(...) should be an options object. See https://github.com/rich-harris/magic-string

### ISSUE THREAD

**Issue by pakastin** [Rich-Harris/butternut#172]
Hi! I tried to squash [this file](https://github.com/pakastin/redom/blob/master/dist/redom.js) and got an error:
> The final argument to magicString.overwrite(...) should be an options object. See https://github.com/rich-harris/magic-string

I tried to run `squash dist/redom.js > dist/redom.min.js` – did I do something wrong?

**Comment by Conduitry** [comment]
Nope, you're not doing anything wrong. The underlying library magic-string changed the arguments to one of its methods. The old way of calling it still works for now, but prints this warning. This has actually already been updated in the master branch of butternut, but it hasn't been released yet.

**Comment by pakastin** [comment]
Oh, I didn't realize it still works – sorry 🤣

**Comment by nifgraup** [comment]
can not reproduce on master

**Comment by lili21** [comment]
when will master be released?

**Comment by pakastin** [comment]
Any news on this..?

**Comment by seltix5** [comment]
hello,
I still get this warning...
thanks

**Comment by DanielRuf** [comment]
> Nope, you're not doing anything wrong. The underlying library magic-string changed the arguments to one of its methods. The old way of calling it still works for now, but prints this warning. This has actually already been updated in the master branch of butternut, but it hasn't been released yet.

Still not released. Is this project still actively maintained including the release process?
